src/queries/index.js

```diff
@@ -75,6 +80,7 @@ export const getQuery = (query, filterOptions = {}) => {
     [QUERY_TYPES.POINTS_OF_INTEREST_AND_TOURS]: GET_POINTS_OF_INTEREST_AND_TOURS,
     [QUERY_TYPES.PUBLIC_HTML_FILE]: GET_PUBLIC_HTML_FILE,
     [QUERY_TYPES.PUBLIC_JSON_FILE]: GET_PUBLIC_JSON_FILE,
+    [QUERY_TYPES.REDEEMED_VOUCHERS]: GET_REDEEMED_VOUCHERS,
```

Comment:
please change it to ´VOUCHERS_REDEEMED` to have all the vouchers at one place

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/c4ed62afe1d53ab40d8f4fad62d781cb09e0613d